gremlin-server/src/main/static/NOTICE

```diff
@@ -64,3 +64,29 @@ HPPC borrowed code, ideas or both from:
 Netty 4.1.86
 ------------------------------------------------------------------------
 Copyright 2014 The Netty Project
+
+------------------------------------------------------------------------
+Objenesis 2.4 (Included through gremlin-shaded)
+------------------------------------------------------------------------
+Copyright 2006-2016 Joe Walnes, Henri Tremblay, Leonardo Mesquita
+
+------------------------------------------------------------------------
+Jackson-Databind 2.14.0 and Jackson-Core (Included through gremlin-shaded)
```

Comment:
I want to push for a slightly less abbreviated version as

> It was originally written by Tatu Saloranta (tatu.saloranta@iki.fi), and has been in development since 2007.
> It is currently developed by a community of developers.

The original notice has a clear intent to give attribution to the community in addition to the original author and I believe we should preserve that. I've updated the notice accordingly (and fixed the ordering). If you still think a shorter version is preferable I can update it again.